descriptors/ModuleDescriptor-template.json

```diff
@@ -65,7 +65,7 @@
       ]
     },
     {
-      "id": "invoice-storage.invoice-number",
+      "id": "invoice-storage.invoice-numbers",
```

Comment:
Please roll-back this interface name back to `invoice-storage.invoice-number` because URL pattern is
`"pathPattern": "/invoice-storage/invoice-number"`